daemon/indexing/listener/listener.js

```diff
@@ -104,10 +96,26 @@ const LISTEN_RULES = {
 // Section 2: The following engine
 // -------------------------------
 
+function setupOriginJS(config){
+  const web3Provider = new Web3.providers.HttpProvider(config.web3Url)
```

Comment:
nit: add a console.log statement to show which provider and ipfs url is being used. Will be useful when checking in production that the listener got started with proper config.

Reply:
Added console.logs for which web3 and which IPFS url are used.